rare-init.sql

```diff
@@ -85,9 +85,11 @@ CREATE TABLE "Categories" (
   "label" varchar
 );
 --
-INSERT INTO Categories ('label')
-VALUES ('News');
+INSERT INTO Categories ("id", "label")
+VALUES (1, "News");
 INSERT INTO Tags ('label')
 VALUES ('JavaScript');
 INSERT INTO Reactions ('label', 'image_url')
-VALUES ('happy', 'https://pngtree.com/so/happy');
\ No newline at end of file
+VALUES ('happy', 'https://pngtree.com/so/happy');
+
+SELECT * FROM Categories;
```

Comment:
This seems extraneous -- could we move this to a separate .sql file like @mecrowley suggested?